packages/core/src/Modules/Cashier/Components/Receipt/account-transfer-receipt.jsx

```diff
@@ -22,17 +27,19 @@ class AccountTransferReceipt extends React.Component {
         return (
             <div className='cashier__wrapper account-transfer__receipt'>
                 <h2 className='cashier__header'>
-                    <Localize i18n_default_text='Your funds have been transferred.' />
+                    <Localize i18n_default_text='Your funds have been transferred' />
```

Comment:
Are we sure we want to remove period | full-stop here? 

Because the full text in other payment transfer receipts have full stop | period. e.g - in `payment-agent-transfer-receipt.jsx` has `Your funds have been transferred to {{name}}.`

Reply:
@zaki-binary yes, we don't use fullstops in header. this PR is just for transfer page, I will slowly redesign the other cashier tabs too after DMT5 changes are done